intellij/src/nl/tudelft/watchdog/intellij/ui/wizards/ProjectRegistrationInputPanel.java

```diff
@@ -62,8 +85,87 @@ private void createInputFields() {
         this.toolsUsed = WizardStep.createLinkedLabelTextField(TOOL_USAGE_LABEL_TEXT, TOOL_USAGE_TEXTFIELD_TOOLTIP, 150, inputContainer);
     }
 
+    private void createSlider(JPanel parent) {
+        JPanel questionPanel = UIUtils.createFlowJPanelLeft(parent);
+
+        UIUtils.createLabel(questionPanel,
+            "Estimate how you divide your time into the two activities testing and production. Just have a wild guess!");
+
+        this.productionPercentageStart = ThreadLocalRandom.current().nextInt(0, 100 + 1);
+
+        JPanel sliderPanel = UIUtils.createFlowJPanelCenter(parent);
+        JPanel row = UIUtils.createFlowJPanelLeft(sliderPanel);
+
+        JLabel testingLabel = UIUtils.createLabel(row, "100% Testing  ");
+        testingLabel
+            .setToolTipText("To the testing activity, everything you do with Junit tests counts. Examples: writing, modifying, debugging, and executing Junit tests");
```

Comment:
This text is different than for Eclipse. If you extract into `WizardStrings` and update the references here, that duplication should be resolved :tada: